### DISCUSSION TITLE
Is this limited to just the example scenarios?

### DISCUSSION THREAD

**Discussion by matari** [melih-unsal/DemoGPT#22] [General]
I can't seem to get it to output the project described in the prompt. It only outputs one of the example scenarios with the name I specified for my application.

**Comment by melih-unsal** [comment]
Hi @matari ,

Really appreciate your attention :blush:   
DemoGPT creates any LLM-based application you prompt, not limited to the examples.
Only the demo application running on Streamlit is limited by the examples which you might see on the website.

To run by your own prompt, you can simply install demogpt by `pip install demogpt` then type `demogpt` on the terminal to run the application. Then, in the application, you can type your idea and demo title and then press the submit button instead of the examples to see the result for your own prompt.

If you face any problem that you prompt, you can write down here then we can easily look over it together.
Waiting for your feedback :pray:

**Comment by matari** [comment]
very strange, I have the package installed and pip lists it as available but it will not run. I checked the PATH. any ideas?

**Comment by melih-unsal** [comment]
Hi @matari , 
Did you type or attach the error message you get so that I can work on it?

**Comment by matari** [comment]
![Screenshot 2023-07-25 153106](https://github.com/melih-unsal/DemoGPT/assets/1245716/74f15446-cd98-4e5e-95fa-06c164d314a5)
here you go. you can see the package is installed and I can see it but it won't run. windows 11 VScode

**Comment by melih-unsal** [comment]
Hi @matari , sorry again.

It seems there is a path issue.where the Python Scripts directory isn't in the system's PATH.
Here's how you can add the Python Scripts directory to your PATH in Windows:

1. Find where your Python Scripts directory is. This will typically be something like C:\Python311\Scripts or C:\Users\YourUsername\AppData\Local\Programs\Python\Python311\Scripts, depending on how Python was installed.
2. Open the start menu, search for Environment Variables, and click on Edit the system environment variables.
3. In the System Properties window that pops up, click on Environment Variables.
4. In the Environment Variables window, under the System variables section, find the Path variable, select it, and click on Edit.
5. In the Edit Environment Variable window, click on New, and then paste the path to the Python Scripts directory.
6. Click OK on all the windows to save these changes.
7. Restart your command prompt to ensure the changes take effect.

If the above solution does not work, you can also try to use DemoGPT with the Source Code usage.

You can do this with these simple steps below:
1. `git clone https://github.com/melih-unsal/DemoGPT.git`
2. `cd DemoGPT`
8.  `pip install -r requirements.txt`
9. `streamlit run src/prompt_based/app.py`

If none of the solutions work, please don't hesitate to write here. :pray:

**Comment by matari** [comment]
![image](https://github.com/melih-unsal/DemoGPT/assets/1245716/2481f536-73a7-48e3-b12c-5800e67eb7c1)
that was the first thing i tried. I have installed the source code but all it produces is one of the examples. can I change that somewhere?

**Comment by melih-unsal** [comment]
Hi @matari , I see,

So it works but it can only generate one of the examples, right?

To clarify the issue, which button do you press to trigger the generation?
**Submit** button or one of the example buttons ?

**Reply by matari** [reply]
I name it (Ctrl+enter), enter the prompt (Ctrl+enter) and hit the Submit button.

**Reply by matari** [reply]
Yes it only generates one of the examples when I hit submit

**Reply by melih-unsal** [reply]
Hi @matari ,
To make sure I'm following the same steps that you've applied to generate the same problem, I need to ask two question.
You prompted **Ctrl+enter** for both title and demo idea, right? If so, did you try on another (title,idea) pair on the application?
Sorry for that again :pray:

**Comment by matari** [comment]
Hey, yes i entered in the name then ctrl+enter then I entered the prompt and ctrl+enter, then i click submit. it seams to ignore the prompt and just create one of the demo apps. I have tried different prompts and its the same behavior

**Comment by melih-unsal** [comment]
Hi @matari ,
Thanks for the clarification. 
Unfortunately, I was not able to simulate what you've gotten.
Could you try without **ctrl+enter**, maybe because it is somehow resetting the text input fields even if I also tried but couldn't see its effect. :pray:
